### PR TITLE
feat(python-stack): Add route generating vegetation difference in an area

### PR DESCRIPTION
**Pull request by FunkySayu**
Implements a route in the Flask API, /forestDiff, getting the vegetation difference between two years in an area. This will return an RGB image where red stands for deforestation between the two years, green stands for reforestation, and blue is an unapplicable part of the image (such as river, ocean...)

### CODE REVIEW

**Comment by FunkySayu** [comment]
TODO fix on this branch the delta time that does not support months < 1 for example.

File: computation/imagefetcher/imagefetcher/fetcher.py [hunk 1]
```diff
@@ -119,3 +168,37 @@ def GetRGBImage(self, start_date, end_date, polygon, scale=100):
...
+        the accuracy is better on this dataset.
+
+        The dataset used to generate this image is the MOD13A1.005 Vegetation
+        Indices 16-Day L3 Global 500m [1], provided publicly by the NASA. It is
+        updated every 16 days, with a maximum scale of 500 meter per pixels.
+        This is the most accurate on the amazon rainforest.
+
+        Parameters:
+            start_year: integer representing the reference year. Must be
+                greater or equal than 2000.
```
Comment (ArchangelX360) [inline computation/imagefetcher/imagefetcher/fetcher.py]: Typo: must be greater **than** or equal **to**.

File: computation/imagefetcher/imagefetcher/app.py [hunk 2]
```diff
@@ -89,57 +81,56 @@ def GetRGBImage():
...
+    See :meth:`ImageFetcher.GetForestIndicesImage` for more informations.
+
+    GET Parameters:
+        polygon (list[list[int]]):
+            Area to visualize. Required.
+        start (int):
+            Reference year. Must be greater or equal than 2000.
+        stop (int):
+            Year on which we will subtract the data generated from start year.
+            Must be greater than start year, and lower than 2016.
```
Comment (ArchangelX360) [inline computation/imagefetcher/imagefetcher/app.py]: Hardcoded 2016 seems a bit weird, why didn't you use a "current year" instead?

File: computation/imagefetcher/imagefetcher/fetcher.py [hunk 3]
```diff
@@ -119,3 +168,37 @@ def GetRGBImage(self, start_date, end_date, polygon, scale=100):
...
+            end_year: integer representing the year on which we will subtract
+                the data generated from the start_year. Must be greater than
+                start_year, and lesser or equal to 2016.
```
Comment (ArchangelX360) [inline computation/imagefetcher/imagefetcher/fetcher.py]: Same comment about 2016.

**Review by ArchangelX360 — CHANGES_REQUESTED** [review]
Need explanation about this hardcoded 2016.

File: computation/imagefetcher/imagefetcher/fetcher.py (hunk 1 above)
Comment (FunkySayu) [reply computation/imagefetcher/imagefetcher/fetcher.py]: Done.

File: computation/imagefetcher/imagefetcher/fetcher.py (hunk 3 above)
Comment (FunkySayu) [reply computation/imagefetcher/imagefetcher/fetcher.py]: Done.

**Review by ArchangelX360 — APPROVED** [review]

**Comment by ArchangelX360** [comment]
LGTM.

**Pull request merged**